test/expected/plan_expand_hypertable-11.out

```diff
@@ -2475,6 +2475,75 @@ SELECT time FROM regular_timestamptz UNION ALL SELECT time FROM metrics_timestam
  Tue Feb 01 00:00:00 2000 PST
 (128 rows)
 
+-- test nested join qual propagation
+:PREFIX
+SELECT * FROM (
+SELECT m1.time FROM metrics_timestamptz m1 FULL OUTER JOIN metrics_timestamptz_2 m2 ON true WHERE m2.time = m1.time AND m1.device_id = m2.device_id AND m2.time < '2000-01-10' AND m1.device_id = 1
+) o1 FULL OUTER JOIN (
+SELECT m1.time FROM metrics_timestamptz m1 FULL OUTER JOIN metrics_timestamptz_2 m2 ON true WHERE m2.time = m1.time AND m1.device_id = m2.device_id AND m2.time > '2000-01-20' AND m1.device_id = 2
+) o2 ON o1.time = o2.time ORDER BY 1,2;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Sort
+   Sort Key: m1."time", m1_5."time"
+   ->  Merge Full Join
+         Merge Cond: (m1_5."time" = m1."time")
+         ->  Nested Loop
+               ->  Merge Append
+                     Sort Key: m2_3."time"
+                     ->  Index Scan Backward using _hyper_7_170_chunk_metrics_timestamptz_2_time_idx on _hyper_7_170_chunk m2_3
+                           Index Cond: ("time" > 'Thu Jan 20 00:00:00 2000 PST'::timestamp with time zone)
+                           Filter: (device_id = 2)
+                     ->  Index Scan Backward using _hyper_7_168_chunk_metrics_timestamptz_2_time_idx on _hyper_7_168_chunk m2_4
+                           Index Cond: ("time" > 'Thu Jan 20 00:00:00 2000 PST'::timestamp with time zone)
+                           Filter: (device_id = 2)
+                     ->  Index Scan Backward using _hyper_7_169_chunk_metrics_timestamptz_2_time_idx on _hyper_7_169_chunk m2_5
+                           Index Cond: ("time" > 'Thu Jan 20 00:00:00 2000 PST'::timestamp with time zone)
+                           Filter: (device_id = 2)
+               ->  Append
+                     ->  Index Scan using _hyper_6_160_chunk_metrics_timestamptz_time_idx on _hyper_6_160_chunk m1_5
```

Comment:
since we have the same tables and identical column names in o1 and o2, I am finding it hard to figure out which join this is.  It would help if you use 2 alases like m3 and m4 to make the plan easier to  follow.